src/app/shared/models/give-help.model.ts

```diff
@@ -0,0 +1,20 @@
+export interface Offer {
+  title: string;
+  description: string;
+  phoneNumber?: string;
+}
+
+export interface Prefix {
+  countryCode: string;
+  prefix: string;
+}
+
+export interface TempTransportOfferDefinitionDTO {
+  title: string;
+  description: string;
+  origin: Location;
+  destination: Location;
+  capacity: number;
+  transportDate: string;
+  phoneNumber?: string;
+}
```

Comment:
Avoid our own types, API already provides them and they're automatically synced to the repo on each `./generate_api.sh` call.

The generated files are in `src/app/core/api` and can be imported from `'@app/core/api'`.

Reply:
TempTransportOfferDefinitionDTO z API nie ma numeru telefonu

Reply:
W ostateczności możesz zdefiniować ten interfejs jako rozszerzający ten z API, dopisując tylko to, czego brakuje. Zakładam, że docelowo to wszystko powinno być; może zgłoś na kanale Backend, jeśli czegoś brakuje, niech dodadzą i możemy potem przegenerować API.

Reply:
Spłaszczyłem komponenty forma w give-help, natomiast utworzyłem plik temporary-consts.ts w shared bo w API nie ma wszystkiego Prefixów. Zamierzam tu wrzucac tymczasowo jakies zamockowane dane czy interfejsy/modele których mi brakuje do czasu aż ogarnie to backend.